### ISSUE TITLE
Cyborg theme will not turn on

### ISSUE THREAD

**Issue by bmorse526** [ntoff/Octoprint-CyborgTheme#16]
Cyborg them is installed and the circle checked under plug in manager.  Theme remains what it is set for in appearance.   How do you turn on the theme?

**Comment by bmorse526** [comment]
Install was done from a smartphone.  Uninstalled on a pc, reo stalled, and theme activated on restart.  Smartphone still only shows standard theme, but pc shows cyborg theme.  Issue is with a mobile browser not the theme.

**Comment by ntoff** [comment]
I would probably use the touchUI plugin for access on your phone, it rearranges some stuff to make octoprint more touch screen (and small screen) friendly (and is also a dark coloured).

In any case, I don't do any kind of browser checking so the cyborg theme won't act differently depending on the device or browser.

I've had issues (not with my theme) with chrome on my phone not refreshing pages and since it's a phone there's no "shift + f5" to force a refresh. Try closing your mobile browser (if your mobile OS supports manually closing apps).

**Comment by bmorse526** [comment]
Thanks for the info.  It works on my pc and that is the main place I wanted
it.
Thank you for creating the plug In

**Comment by ntoff** [comment]
Also something I forgot to mention from your original question, octoprint's "appearance" section pretty much only applies to the colour of the nav bar (how I got the purple nav bar in the screenshot of this theme), no other colours are changed by that. The only controls this theme has is an on/off switch in the plugin manager since OctoPrint doesn't officially support themes.

I share your confusion, I made the plugin and still I keep checking the "appearance" section to turn it on and off, that seems like the logical place for it! But no, as long as it's on in the plugin manager, it should be active.
